pkgs/tools/misc/man-db/default.nix

```
@@ -1,17 +1,32 @@
-{ lib, stdenv, fetchurl, pkg-config, libpipeline, db, groff, libiconv, makeWrapper, buildPackages }:
+{ buildPackages
+, db
+, fetchurl
+, gettext
+, groff
+, lib
+, libiconv
+, libpipeline
+, libtool
+, makeWrapper
+, pkg-config
+, stdenv
+, zstd
+, autoreconfHook
+}:
 
 stdenv.mkDerivation rec {
-  name = "man-db-2.9.4";
+  pname = "man-db";
+  version = "2.9.4";
 
   src = fetchurl {
-    url = "mirror://savannah/man-db/${name}.tar.xz";
+    url = "mirror://savannah/man-db/${pname}-${version}.tar.xz";
     sha256 = "sha256-tmyZ7frRatkoyIn4fPdjgCY8FgkyPCgLOp5pY/2xZ1Y=";
   };
 
   outputs = [ "out" "doc" ];
   outputMan = "out"; # users will want `man man` to work
 
-  nativeBuildInputs = [ pkg-config makeWrapper groff ];
+  nativeBuildInputs = [ autoreconfHook gettext groff libtool makeWrapper pkg-config zstd ];
```

```suggestion
  nativeBuildInputs = [ autoreconfHook groff makeWrapper pkg-config zstd ];
```

they are already included in the hook.